### ISSUE TITLE
"ADDJOB" returns a redis status reply, which gets turned into a ruby symbol.

### ISSUE THREAD

**Issue by Asmod4n** [matsumoto-r/mruby-disque#1]
Have just played around with disque, "ADDJOB" returns a redis status reply which gets turned into a ruby symbol. Symbols are not garbage collected in mruby so it allocates memory till nothing is left when this gem is used for a long time :/

**Comment by matsumotory** [comment]
@Asmod4n Yes. So should we change the mrb_value type in [src/mrb_redis.c](https://github.com/matsumoto-r/mruby-redis/blob/master/src/mrb_redis.c#L908-L911)?

**Comment by Asmod4n** [comment]
@matsumoto-r 
Maybe it's time to add symbol GC to mruby, a change to mrb_str_new should work till then but will break tests and the API.
